cartridges/int_alma/cartridge/scripts/helpers/almaHelpers.js

```diff
@@ -160,6 +160,22 @@ function haveExcludedCategory(productIds) {
     return haveExcludedCategoryReturn;
 }
 
+/**
+ * Get the full url for a page
+ * @param {string} pageName name of page for url
+ * @param {string} pageTemplate template of page for url
+ * @param {string} locale locale
+ * @returns {string} url
+ */
+function getFullPageUrl(pageName, pageTemplate, locale) {
+    var hostname = Site.getCurrent().getHttpsHostName();
+    var siteName = Site.getCurrent().getName();
+
+    var url = 'https://' + hostname + '/s/' + siteName + '/' + pageName + '/' + pageTemplate + '.html?lang=' + locale;
```

Comment:
return 'https://' + hostname + '/s/' + siteName + '/' + pageName + '/' + pageTemplate + '.html?lang=' + locale; instead of put it in a variable and return it